SonarQube.MSBuild.Tasks/SonarQube.MSBuild.Tasks.csproj

```diff
@@ -1,16 +1,78 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Why reverting the use of the new csproj format? It is supposed to be handled by net46.

Reply:
Good tip! I've moved it back to the future,